tests/test_note_array.py

```diff
@@ -175,9 +175,9 @@ def test_ensure_na_different_divs(self):
         # note_arrays = [p.note_array(include_divs_per_quarter= True) for p in parts]
         merged_note_array = ensure_notearray(parts)
         for note in merged_note_array[-4:]:
-            self.assertTrue(note["onset_div"] == 92)
-            self.assertTrue(note["duration_div"] == 4)
-            self.assertTrue(note["divs_pq"] == 4)
+            self.assertTrue(note["onset_div"] == 2208)
```

Comment:
Not sure if it was necessary to change the test values?

Reply:
yes, new import changed order of notes.